rules.json

```
@@ -1,17 +1,170 @@
+    "• Do NOT tag (using @ to ping or send a notification to a specific member) any staff members unless you have a valid reason to do so. Also, do not tag any specific users repeatedly.",
+    "• It is under your discretion to share your own personal information with other members such as race, sex, sexual orientation, financial status, full names, addresses, IP’s, appearances, and voices. However, do NOT share others’ personal information, unless given permission.",
+    "• For the sake of everyone, it is only fair and respectful that TXNRP is an ENGLISH only network. Please refrain from speaking any other languages than English in any text/voice channels.",
+    "Advertising",
+    "• Do NOT have a social media link, IP, or another FiveM server in your discord status message.",
+    "• Do NOT advertise another server in our text/voice channels. You may advertise another server in Private Direct Messages, but ONLY IF THEY ASK FOR IT. Failure to follow this rule will result in a permanent ban.",
+    "• Similarly, do not advertise your YouTube channel or Social Media to any users unless they ASK FOR IT.",
+    "Notes",
+    "Anything you say to users through private messages on any application can be punishable on the Discord Server as well. That includes harassment, unnecessary swearing, sending random links, etc. Message a staff member with solid evidence, such as screenshots, videos, etc, if you feel someone should be punished for something they said through Private Messages.",
+
+    "We, The TXDPSRP Network also strictly follow the Discord Terms of Service (TOS) and their Guidelines! If you are found breaking either of these you will be BANNED without notice and/or explanation"
+        ]
+            
+
+
+
+
+
+
+            
+
     },
     {
-        "title": "Roleplay Standards",
-        "isChild": true,
+        "title": "Miscellaneous Rules",
+        "isChild": false,
         "rules": [
-            "You must realistically roleplay at all times. Fail-RolePlay (FailRP), Random Deathmatches (RDM), Vehicle Deathmatches (VDM), Meta-Gaming, Noclipping, Teleporting, Godmode, Invisibility and/or any type of so-called “Trolling” or “No Intent to RP (NITRP)” will not be tolerated.",
-            "Do NOT intentionally bait Law Enforcement Officers, also known as Copbaiting; for example, do NOT try to intentionally get a Law Enforcement Officer to initiate a traffic stop on you.",
-            "Do NOT interfere with any roleplay that you are not apart of.",
-            "Do NOT perform any Erotic RP. This includes roleplay containing sexism and nudity.",
-            "Whatever else would go here, this is just an example...."
+     "• Do NOT sell or purchase anything of some sort that displays the TXNRP Logo, unless approved, distributed, and/or promoted by Head Administration.",
+     "• Do NOT create any discord/teamspeak servers in the name of TXNRP, unless approved by the founder.",
```

We no longer use teamspeak in any capacity 
```suggestion
     "• Do NOT create any discord servers in the name of TXNRP, unless approved by the founder.",
```